lib/src/analyzers/lint_analyzer/lint_config.dart

```diff
@@ -1,27 +1,63 @@
-import 'package:glob/glob.dart';
 import 'package:meta/meta.dart';
 
-import 'anti_patterns/models/obsolete_pattern.dart';
-import 'metrics/models/metric.dart';
-import 'rules/models/rule.dart';
+import '../../cli/models/parsed_arguments.dart';
+import '../../config_builder/analysis_options_utils.dart';
+import '../../config_builder/models/analysis_options.dart';
+import 'metrics/metrics_factory.dart';
 
+/// Class representing config
```

Comment:
@incendial please add information about this class is optimized for manipulation like merge

Reply:
Done.